### DISCUSSION TITLE
Récupération du token Enedis

### DISCUSSION THREAD

**Discussion by edwinmarcellin** [bokub/conso-api#24] [Général]
Bonjour, 

J'ai validé l'accès aux données Enedis, mais j'ai perdu le token et ne peut pas le récupérer. 

Auriez vous une solution pour le récupérer.
Merci

**Comment by bokub** [comment]
En régénérer un nouveau ?

N.B le site d'enedis est cassé en ce moment, il peuvent dire que ton compteur est incompatible.

Il faut simplement attendre qu'ils réparent, cf #26
